sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/unresolved.scala

```diff
@@ -234,7 +235,7 @@ case class UnresolvedGenerator(name: FunctionIdentifier, children: Seq[Expressio
   override def nullable: Boolean = throw new UnresolvedException("nullable")
   override lazy val resolved = false
 
-  override def prettyName: String = name.unquotedString
+  override def prettyName: String = name.unquotedString(SESSION_CATALOG_NAME)
   override def toString: String = s"'$name(${children.mkString(", ")})"
```

Comment:
This will include the unquoted identifier *without* the catalog, as the `toString` method of `FunctionIdentifier` is implemented as `unquotedString`. Is this intentional?